src/openepd/model/org.py

```diff
@@ -51,14 +51,14 @@ class Org(WithAttachmentsMixin, WithAltIdsMixin, OrgRef):
     # TODO: NEW field, not in the spec
 
     owner: Optional["Org"] = pyd.Field(description="Organization that controls this organization", default=None)
-    subsidiaries: Annotated[list[str], pyd.conlist(pyd.constr(max_length=200), max_items=255)] | None = pyd.Field(
+    subsidiaries: Annotated[list["Org"], pyd.conlist(pyd.constr(max_length=200), max_items=255)] | None = pyd.Field(
```

Comment:
May be `OrgRef`?

Reply:
OrgRef is also OK for me - it is smaller. 
agreed?

Reply:
Discussed with Axe - agreed.